3-0-libraries.md

```diff
@@ -149,6 +149,25 @@ To change many libraries:
 1. Repeat for all the libraries.
 1. Click _Save_.
 
+## How can I delete a library?
+Libraries can be deleted from the Libraries list page by selecting the libraries and clicking _Delete_. A
+library cannot be deleted if it has any dilutions. Only the library's creator or an admin can delete a library.
+Deleted items are listed on the Deletions list page.
+
+## How can I delete a dilution?
+Dilutions can be deleted from the Dilutions list page by selecting the dilutions and clicking _Delete_. A
+dilution cannot be deleted if it has been added to any pools. Only the dilution's creator or an admin can
+delete a dilution. Deleted items are listed on the Deletions list page.
+
+## How can I delete a pool?
+Pools can be deleted from the Pools list page by selecting the pools and clicking _Delete_. A pool cannot
+be deleted if it has been added to any partitions. Any associated orders will be deleted along with a pool.
+Only the pool's creator or an admin can delete a pool. Deleted items are listed on the Deletions list page.
+
+## How can I delete an order?
+Orders can be deleted from the Edit Pool page by selecting the orders and clicking _Delete_. A
+library cannot be deleted if it has any dilutions. Deleted items are listed on the Deletions list page.
+
```

Comment:
This should probably go in the `_includes/libraries-trouble.md` page, which is included on both plain and detailed libraries pages. (`How can I add a new targeted sequencing type, kit, or anything else in drop-down menus?` and `How are matrix tube barcodes assigned to tubes?` sections from `3-0-libraries.md` could also be included in that file, but it's not urgent.)